### ISSUE TITLE
[Bug] specifying a workspace on command line does not override `default_workspace`

### ISSUE THREAD

**Issue by gbcreation** [nvim-telescope/telescope-frecency.nvim#265]
Hi! :wave: 

When setting `default_workspace = "CWD"` and a list of `workspaces` like: 

```lua
    workspaces = {
      ["conf"] = "/home/my_username/.config",
      ["data"] = "/home/my_username/.local/share",
      ["project"] = "/home/my_username/projects",
      ["wiki"] = "/home/my_username/wiki",
    }
```

Running `:Telescope frecency workspace=conf` displays files from both `/home/my_username/.config` and the current working folder. Files from the `conf` workspace are listed first, then files from CWD. It's easier to see by entering some input to reduce the number of listed items (if the input matches files from both folders of course! :) ).

As `workspace=conf` is specified on the command line, I would expect that only files from the conf workspace would be listed, and the `default_workspace` setting would apply only when no workspace option is used. Am I right?

**Comment by delphinus** [comment]
@gbcreation Thank you for report! I solved this in #267. Check it.

**Comment by gbcreation** [comment]
Awesome! Thank you so much :pray:
